torch/csrc/utils/throughput_benchmark.cpp

```diff
@@ -39,7 +39,7 @@ py::object ThroughputBenchmark::runOnce(py::args&& args, py::kwargs&& kwargs) {
   }
 }
 
-ThroughputBenchmark::ThroughputBenchmark(jit::Module script_module)
+ThroughputBenchmark::ThroughputBenchmark(const jit::Module& script_module)
     : script_module_(script_module) {}
 
```

Comment:
```suggestion
ThroughputBenchmark::ThroughputBenchmark(jit::Module script_module)
    : script_module_(std::move(script_module)) {}
```
pass by value is usually more useful for ctors.